packages/dds/tree/src/feature-libraries/sequence-field/sequenceFieldToDelta.ts

```diff
@@ -73,7 +88,13 @@ export function sequenceFieldToDelta<TNodeChange>(
 						moveId: brandOpaque<Delta.MoveId>(mark.id),
 						count: mark.count,
 					};
-					out.pushContent(moveMark);
+					markList.pushContent(moveMark);
+					if (mark.changes !== undefined) {
```

Comment:
It would be nice if we didn't have this logic duplicated across different mark types, since it seems like there are only two cases: input spanning marks, and non-input spanning marks. Maybe this code will be removed soon, in which case don't worry about it.